scripts/xtensa-build-all.sh

```diff
@@ -219,10 +245,16 @@ do
 		tgl)
 			PLATFORM="tigerlake"
 			ARCH="xtensa-smp"
-			XTENSA_CORE="X6H3CNL_2017_8"
+			XTENSA_CORE="cavs2x_LX6HiFi3_2017_8"
 			HOST="xtensa-cnl-elf"
 			XTENSA_TOOLS_VERSION="RG-2017.8-linux"
 			HAVE_ROM='yes'
+			if [ -z "$PRIVATE_KEY_OPTION" ]
+			then
+				PRIVATE_KEY_OPTION="-D${SIGNING_TOOL}_PRIVATE_KEY"
+				PRIVATE_KEY_OPTION="$PRIVATE_KEY_OPTION=$pwd/keys"
+				PRIVATE_KEY_OPTION="$PRIVATE_KEY_OPTION/otc_private_key_3k.pem"
+			fi
```

Comment:
You make this default value specific to tgl. Shouldn't it be (less) specific to MEU instead? Either way a short comment about this key wouldn't hurt.

Reply:
> You make this default value specific to tgl. Shouldn't it be (less) specific to MEU instead?

I got that wrong, it's the other way round. `otc_private_key_3k.pem` is (obviously?) specific to TGL, in theory it doesn't not depend on which signing tool is used. Granted: one tool supports TGL and the other not yet but that's not something this part of the code must know about. The TGL+rimage situation is already dealt with at the bottom of this patch, no need to pollute this part too.

Long story short please remove the `[ -n "${MEU_TOOL_PATH}" ]` part of the condition.